src/main/java/org/opentestsystem/delivery/logging/EventParser.java

```diff
@@ -35,7 +36,7 @@
    */
   public Optional<EventInfo> parsePostHandle(final HttpServletRequest request, final HttpServletResponse response, final Object
     handler, EventLogger logger) {
-    return Optional.of(EventInfo.create(request.getPathInfo(), getEventDataFields(request)));
+    return Optional.of(EventInfo.create(StringUtils.trimToEmpty(request.getPathInfo()), getEventDataFields(request)));
```

Comment:
These EventParser methods were intended to be overridden. They are not required to return an event to log.